python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -27,13 +29,20 @@
 
 from cugraph.dask.comms import comms as Comms
 
+from typing import Sequence, List, Union, Tuple
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from cugraph import Graph
+
 src_n = "sources"
```

Comment:
no code change

We really need a utility class with all the standard column names.